8Knot/pages/repo_overview/visualizations/package_version.py

```diff
@@ -124,21 +124,31 @@ def process_data(df: pd.DataFrame):
     # get max time difference to use for grouping and round for correct format
     max_time = round(df["libyear"].max(), 3)
 
-    # group by the 4 buckets
-    depend_pie = pd.DataFrame(
-        df["libyear"].groupby(pd.cut(df["libyear"], [0, 0.001, 0.5, 1, max_time], right=False)).count()
-    ).rename(columns={"libyear": "packages"})
+    # create intervals for df grouper
+    intervals = [0.0, 0.001, 0.5, 1.0, max_time]
```

Comment:
What does a 0.0 vs. a 0.001 interval mean? I'm guessing that 0.5 means 6 months, and 1.0 means a year, but I'm unclear on why this is the case.

Reply:
@cdolfi

Reply:
That is pretty much a hack of getting a bucket just for "up to date" if you do 0,0.0 it wont include anything, by giving a very minor increment larger it will allow for all '0' values to be put in that group